src/steps/os/unix.rs

```diff
@@ -282,6 +301,25 @@ pub fn run_brew_formula(ctx: &ExecutionContext, variant: BrewVariant) -> Result<
         }
     }
 
+    #[cfg(target_os = "linux")]
+    {
+        let sudo_uid = brew_linux_sudo_uid();
+        // if brew is owned by another user, execute "sudo -Hu <uid> brew update"
+        if let Some(uid) = sudo_uid {
+            let sudo_uid_str = sudo_uid.map_or_else(|| "".to_string(), |uid| uid.to_string());
+            print_separator(format!("{} (sudo as {})", variant.step_title(), sudo_uid_str));
```

Comment:
Let's use the user name rather than ID here:

```suggestion
            let user_name = nix::unistd::User::from_uid(uid)
                .expect("failed to call getpwuid()")
                .expect("this user should exist");
            print_separator(format!("{} (sudo as {})", variant.step_title(), user_name));
```